packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx

```diff
@@ -427,7 +427,7 @@ const Sidebar = (): JSX.Element => {
               type: 'action',
             })
           }
-          href={'#/catalog'}
+          href={navigationPlugin.getURL('catalog', '/')}
```

Comment:
Dans l'ancienne [PR](https://github.com/ovh/manager/pull/10567/files) que tu viens d'update c'était sa , tu es sur de la modif ? 

```suggestion
    navigationPlugin.getURL('hub', '#/catalog')
```

Reply:
catalog is a standalone app (not embedded in hub app) now so the current link is ok

Reply:
Yes, url is inside the shell to be more secure